app/mailers/application_mailer.rb

```diff
@@ -1,4 +1,4 @@
 class ApplicationMailer < ActionMailer::Base
-  default from: "workshop.portal@gmail.com"
+  default from: Rails.configuration.default_reply_to_email
```

Comment:
Right now, we can access the sender adress with ENV['gmail_username'].
It returns just the name before the @ tho